### PR TITLE
docs: make it clearer that posix mode must be used in WSL

### PR DESCRIPTION
**Pull request by yunchiri**
clearly tell to person what select mode

### CODE REVIEW

File: doc/build-windows.md [hunk 1]
```diff
@@ -77,9 +77,10 @@ The first step is to install the mingw-w64 cross-compilation tool chain:
 
     sudo apt install g++-mingw-w64-x86-64
 
-Ubuntu Bionic 18.04 <sup>[1](#footnote1)</sup>:
+Ubuntu Bionic 18.04 <sup>[1](#footnote1)</sup> (select posix mode):
```
Comment (practicalswift) [inline doc/build-windows.md]: Trailing whitespace :-)

File: doc/build-windows.md [hunk 2]
```diff
@@ -77,9 +77,10 @@ The first step is to install the mingw-w64 cross-compilation tool chain:
 
     sudo apt install g++-mingw-w64-x86-64
 
-Ubuntu Bionic 18.04 <sup>[1](#footnote1)</sup>:
+Ubuntu Bionic 18.04 <sup>[1](#footnote1)</sup> (select posix mode): 
 
-    sudo update-alternatives --config x86_64-w64-mingw32-g++ # Set the default mingw32 g++ compiler option to posix.
+    sudo update-alternatives --config x86_64-w64-mingw32-g++ # Set the default mingw32 g++ compiler option to posix
+
```
Comment (practicalswift) [inline doc/build-windows.md]: Nit: Multiple consecutive blank lines.

**Comment by fanquake** [comment]
Are the comments behind each `update-alternatives` instruction, plus the [footnote](https://github.com/bitcoin/bitcoin/blob/0811e14cabe279c2f8363c76db06ddd5a0c856ed/doc/build-windows.md#footnotes) not enough?

If you are going to make these changes, you should do it for the `32 bit` instructions as well. 
Also needs a proper commit message (squash any fix-ups into a single commit).

**Comment by yunchiri** [comment]
@fanquake 
as newbie, footnote is not enough get information before click
I think tell what select clearly!
Thanks feedback

**Comment by fanquake** [comment]
@yunchiri If you are going to make the updates, you can reopen and push changes to this Pull Request, you don't have to open a new one.

**Pull request closed without merging**